opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -337,35 +337,42 @@ impl BatchLogProcessor {
                     match message_receiver.recv_timeout(remaining_time) {
                         Ok(BatchMessage::ExportLog(log)) => {
                             logs.push(log);
-                            if logs.len() == config.max_export_batch_size
-                                || last_export_time.elapsed() >= config.scheduled_delay
```

Comment:
time-triggered export is already handled by the overall loop, not need to repeat.